Mathlib/Analysis/NormedSpace/SphereNormEquiv.lean

```diff
@@ -38,5 +38,9 @@ noncomputable def homeomorphUnitSphereProd :
     rw [mem_Ioi] at hr
     ext <;> simp [hx, norm_smul, hr.le, abs_of_pos hr, hr.ne']
   continuous_toFun := by
-    refine .prod_mk (.codRestrict (.smul (.inv₀ ?_ ?_) ?_) _) ?_ <;> continuity
-  continuous_invFun := by continuity
+    refine .prod_mk (.codRestrict (.smul (.inv₀ ?_ ?_) ?_) _) ?_
+    · fun_prop
+    · continuity
+    · fun_prop
+    · apply Continuous.subtype_mk (by fun_prop)
```

Comment:
It would be nice to investigate why Continuous.subtype_mk (which is tagged fun_prop) doesn't fire here.

Reply:
```suggestion
    · fun_prop
```
works just fine, right?

Reply:
Oh, indeed it works here; changed. (It does not work on the next line.)